scripts/ci/debian/mono-llvm.install

```diff
@@ -0,0 +1,1574 @@
+usr/lib/libLLVMAMDGPUAsmParser.a
```

Comment:
debian/install allows wildcards and directories, which might make this whole thing a bit more sustainable.

You don't need to worry about including things that are from elsewhere - the entire build root has been `make install DESTDIR=debian/tmp/`'d so the whole prefix is entirely dedicated to this package. If you want EVERYTHING, you could simplify this entire file down to just `usr`